src/components/common/user/UserIcon.tsx

```diff
@@ -12,12 +12,12 @@ import { AppContext, useClient } from "../../../context/revoltjs/RevoltClient";
 
 import IconBase, { IconBaseProps } from "../IconBase";
 import fallback from "../assets/user.png";
+import {VolumeMute} from "@styled-icons/boxicons-solid";
 
-type VoiceStatus = "muted";
 interface Props extends IconBaseProps<User> {
     mask?: string;
     status?: boolean;
-    voice?: VoiceStatus;
+    voice?: string;
```

Comment:
Why is this type becoming ambiguous?
I don't see why it can't just be "muted" | "deaf".

Reply:
It was adding more complexity than needed (in my eyes).

The same functionality can be done with a string and a smaller final package size